flow/ApiVersions.cmake

```diff
@@ -1,6 +1,6 @@
 # API Versions
-set(FDB_AV_LATEST_VERSION                   "730")
-set(FDB_AV_LATEST_BINDINGS_VERSION          "720")
+set(FDB_AV_LATEST_VERSION                   "740")
```

Comment:
In the Snowflake versioning scheme, it should be 710300 (see the release-71.2 branch for comparison). I would set bindings version to the the same version: 710300. I am not sure why Apple kept it lagging behind in main, but for the release we should be the same.

Reply:
Done.